modules/upnp/doc_classes/UPNP.xml

```diff
@@ -7,7 +7,7 @@
 		This class can be used to discover compatible [UPNPDevice]s on the local network and execute commands on them, like managing port mappings (for port forwarding/NAT traversal) and querying the local and remote network IP address. Note that methods on this class are synchronous and block the calling thread.
 		To forward a specific port (here [code]7777[/code], note both [method discover] and [method add_port_mapping] can return errors that should be checked):
 		[codeblock]
-		var upnp = UPNP.new()
+		var upnp := UPNP.new()
```

Comment:
```suggestion
		var upnp = UPNP.new()
```
Generally the documentation uses dynamically typed code, see [here](https://docs.godotengine.org/en/stable/contributing/documentation/docs_writing_guidelines.html#dynamic-vs-static-typing), so I wouldn't add static typing features to this example